src/encoded/audit/file.py

```diff
@@ -335,6 +335,27 @@ def audit_duplicate_quality_metrics(value, system):
                 level='INTERNAL_ACTION'
             )
 
+def audit_file_output_category(value, system):
+    output_category = value.get('output_category')
+    file_format = value.get('file_format')
+    file_id = value.get('@id')
+    raw_file_formats = ['rcc', 'idat', 'CEL', 'csfasta', 'fastq', 'csqual',
+                        'sra']
+    # These three formats can be either raw or processed, so ignore them
+    if file_format in ['tar', 'tsv', 'fasta']:
+        return
+    if file_format in raw_file_formats and output_category != 'raw data':
+        detail = 'File {} has file format {} '.format(file_id, file_format) + \
```

Comment:
I realize this is done throughout our code base so it's a copy/paste job, but it's not good string concatenation.   Try to update these issues as you change the lines.  Using slash for line continuation is an old style, parens are current standards for multi line strings.  Also we only need one format call.
```
        detail = (
            'File {} has file format {} corresponding to raw data but '
            'has an output category of {}'.format(
                file_id,
                file_format,
                output_category,
            )
        )
``` 
Please fix here and on the other one below.